docs/send-data/opentelemetry-collector/time-reference.md

```diff
@@ -0,0 +1,73 @@
+---
+id: time-reference
+title: OTel Timestamp and Date Format Reference
+description: Learn about timestamp formats recognized by the Sumo Logic OpenTelemetry (OTel) Collector and available configuration options.
+keywords:
+  - time-stamp
+  - otel-collector
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::info
+This page specifically covers the timestamp formats recognized by the Sumo Logic OpenTelemetry (OTel) Collector. While Sumo Logic’s [general time reference documentation](/docs/send-data/reference-information/time-reference/) explains how timestamps are parsed and indexed across all data sources, this guide focuses on the formats automatically detected and parsed by the OTel collector for log data.
```

Comment:
This page lists the timestamp formats recognized by the Sumo Logic OpenTelemetry (OTel) Collector. For general information on how Sumo Logic parses and indexes timestamps across all data sources, see the [general time reference documentation](/docs/send-data/reference-information/time-reference).